kingpin/actors/base.py

```diff
@@ -488,7 +488,10 @@ def execute(self):
             raise gen.Return()
 
         try:
-            result = yield self.timeout(self._execute)
+            if hasattr(self, '_execution_timeout'):
```

Comment:
I do not want to muddy up the base actor with information that is relevant only for a particular type of actual actor - ie, `_execution_timeout` is a method only defined in the cloudformation.py file, so the `base.py` file should know nothing about it.